src/ReportTransaction.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace MaxMind;
+
+use MaxMind\Exception\InvalidInputException;
+
+class ReportTransaction extends ServiceClient
+{
+    /**
+     * @param int    $accountId  Your MaxMind account ID
+     * @param string $licenseKey Your MaxMind license key
+     * @param array  $options    An array of options. Possible keys:
+     *
+     * * `host` - The host to use when connecting to the web service.
+     * * `userAgent` - The prefix for the User-Agent header to use in the
+     *   request.
+     * * `caBundle` - The bundle of CA root certificates to use in the request.
+     * * `connectTimeout` - The connect timeout to use for the request.
+     * * `timeout` - The timeout to use for the request.
+     * * `proxy` - The HTTP proxy to use. May include a schema, port,
+     *   username, and password, e.g., `http://username:password@127.0.0.1:10`.
+     * * `validateInput` - Default is `true`. Determines whether values passed
+     *   to the `with*()` methods are validated. It is recommended that you
+     *   leave validation on while developing and only (optionally) disable it
+     *   before deployment.
+     */
+    public function __construct(
+        $accountId,
+        $licenseKey,
+        $options = []
+    ) {
+        parent::__construct($accountId, $licenseKey, $options);
+    }
+
+    /**
+     * @param array $values the transaction parameters
+     *
+     * @throws InvalidInputException   when the request has missing or invalid
+     *                                 data
+     * @throws AuthenticationException when there is an issue authenticating the
+     *                                 request
+     * @throws InvalidRequestException when the request is invalid for some
+     *                                 other reason, e.g., invalid JSON in the POST.
+     * @throws HttpException           when an unexpected HTTP error occurs
+     * @throws WebServiceException     when some other error occurs. This also
+     *                                 serves as the base class for the above exceptions.
+     */
+    public function reportTransaction($values)
```

Comment:
This stutters a bit `ReportTransaction->new(..)->reportTransaction`. That said, I see why you picked it given the minFraud ones are named after the endpoint. Maybe `report` would be sufficient? Not sure.

Reply:
How about `sendReport`?

Reply:
If we are going to break consistency, which I think is fine here, how about just `send`?

Reply:
I've gone back to `report` as that's the operative verb in the API's name. I hope that's OK.